assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -81,7 +82,11 @@ export default function WidgetAreaRenderer( { slug, totalAreas } ) {
 	// Returning `null` here however would have the side-effect of making
 	// all widgets active again, which is why we must return the "null" output.
 	if ( ! activeWidgets.length ) {
-		return widgetsOutput;
+		return (
+			<Grid className={ `${ HIDDEN_CLASS } googlesitekit-widget-area googlesitekit-widget-area--${ slug } googlesitekit-widget-area--${ style }` }>
```

Comment:
this isn't very DRY as it copies a lot of line 95.  It's only repeated twice though, so not enough to be worth extracting out and creating a new component.

Alternatively, I could have this conditionality below on line 96 but IMHO it starts to get really complicated in React when doing this. Better to repeat the parent wrapper component

Reply:
This is mostly the same as the line below as you said, but let's update these to use `classnames` for readability rather than the long template string. E.g. https://github.com/google/site-kit-wp/blob/97f97ffd4e99e8381030b95df2488393f5e70276/assets/js/components/ChangeArrow.js#L10-L12

Reply:
Classnames used @aaemnnosttv :+1:

Reply:
Sorry if this wasn't clear before, but please update both instances (hidden and non-hidden grids) since these are more/less the same.

Reply:
Done @aaemnnosttv